assignment-2/cmmi.md

```diff
@@ -1,58 +1,164 @@
-| **Maturity Level 5 - Optimizing**                    |
-| CAR - Causal Analysis and Resolution         | Not at all | Partially | |
-| OPM - Organizational Performance Management  | Partially | Not at all | Not at all |
+| CM - Configuration Management                | Not at all | Fully      | Unknown    |
+| MA - Measurement and Analysis                | Partially  | Partially  | Not at all |
+| PMC - Project Monitoring and Control         | Partially  | Fully      | Not at all |
+| PP - Project Planning                        | Fully      | Partially  | Not at all |
+| PPQA - Process and Product Quality Assurance | Partially  | Fully      | Partially  |
+| REQM - Requirements Management               | Fully      | Fully      | Not at all |
+| SAM - Supplier Agreement Management          | Not at all | Not at all | Partially  |
+| **Maturity Level 3 - Defined**                                                      |
+| DAR - Decision Analysis and Resolution       | Not at all | Not at all | Unknown    |
+| IPM - Integrated Project Management          | Partially  | Fully      | Partially  |
+| OPD - Organizational Process Definition      | Not at all | Not at all | Not at all |
+| OPF - Organizational Process Focus           | Partially  | Partially  | Not at all |
+| OT - Organizational Training                 | Not at all | Not at all | Unknown    |
+| PI - Product Integration                     | Partially  | Fully      | Not at all |
+| RD - Requirements Development                | Partially  | Fully      | Unknown    |
+| RSKM - Risk Management                       | Not at all | Fully      | Not at all |
+| TS - Technical Solution                      | Partially  | Partially  | Partially  |
+| VAL -  Validation                            | Partially  | Fully      | Partially  |
+| VER -  Verification                          | Partially  | Fully      | Partially  |
+| **Maturity Level 4 - Quantitatively Managed**                                       |
+| OPP - Organizational Process Performance     | Partially  | Not at all | Unknown    |
+| QPM - Quantitative Project Management        | Not at all | Not at all | Unknown    |
+| **Maturity Level 5 - Optimizing**                                                   |
+| CAR - Causal Analysis and Resolution         | Not at all | Partially  | Unknown    |
+| OPM - Organizational Performance Management  | Partially  | Not at all | Not at all |
 
 ## Scrum
 
 | Cmmi Process Area | Covered | Rationale  |
 | ------------------| ------- | ---------  |
 | **Level 2**           |
-| CM                | Not at all | Scrum does not offer any information on the configuration management of work products.            |
+| CM                | Not at all | There is no explicit mention of this in Scrum. CM is important to software development thus this leads to it becoming the task of Scrum team having to figure out how to perform it.             |
```

Comment:
CM can be bigger that one team though, so I'd argue that its bigger than scrum instead of stating that its their responsibility